consoles/sshVirtsh.pm

```diff
@@ -488,6 +488,7 @@ sub add_disk ($self, $args) {
     return;
 }
 
+# uncoverable block
```

Comment:
What for? Shouldn't be so hard to just cover some variable settings. Just call it in tests.

Reply:
hard maybe not. but it is not essential IMO. What do you really get from those test?

Reply:
We should cover everything, as long as it's possible and feasible.
Covering code in tests ensures that the code doesn't break.
If we start discussing about which functions are too simple to be broken, we're going down a rabbit hole, and should rather use the time to just cover the code.

Code coverage does not guarantee 100% that code will work for any case, but it is *much* better than uncovered code.